.github/workflows/check-duplicate-images.yml

```diff
@@ -40,9 +40,16 @@ jobs:
         uses: peter-evans/create-pull-request@v6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          committer: SSW-Rules-Bot <rules-bot@example.com>
+          author: SSW-Rules-Bot <rules-bot@example.com>
           commit-message: Auto Rename Duplicate Images
           title: Auto Rename Duplicate Images
-          body: Generated Automatically via GitHub Actions
+          body: |
+            Generated Automatically via GitHub Actions.
+
+            This PR was created because automated checks found duplicate image names in Rules.Content and tried to rename them.
```

Comment:
```suggestion
            This PR was created because an automated check found multiple images with the same name in SSW.Rules.Content and tried to rename them.
```